sql2.txt

```diff
@@ -2,51 +2,84 @@
 to join the OrderDetails table with the product table.  For TotalCount, you will Use
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
+SELECT Products.ProductName, SUM(OrderDetails.Quantity) AS TotalCount
+FROM Products
+    JOIN OrderDetails ON Products.ProductID = OrderDetails.ProductID
+    GROUP BY Products.ProductID, Products.ProductName
+    ORDER BY Products.ProductName ASC;
 
 
 2. Retrieve the customer name and OrderDetailsID for all the OrderDetails for each customer,
 ordered by customer name. Paste your SQL statement below. You will have to join the Customers
 table with the Orders table, and the Orders table with the OrderDetails table, 
 because the Orders table is the one that ties the Customer table to the OrderDetails table.
-
+SELECT Customers.CustomerName, OrderDetails.OrderDetailID
+FROM OrderDetails
+    JOIN Customers on Orders.CustomerID = Customers.CustomerID
+    JOIN Orders on OrderDetails.OrderID = Orders.OrderID
+    GROUP BY Customers.CustomerName, OrderDetails.OrderDetailID 
```

Comment:
Since there's no aggregate function (like `SUM` in the previous answer), there's actually no benefit in grouping at all; it just makes the database work more.

```suggestion
```